src/views/ReviewConfirm.vue

```diff
@@ -76,8 +83,15 @@ export default class ReviewConfirm extends Mixins(ResourceLookupMixin) {
 </script>
 
 <style lang="scss" scoped>
-.step-container {
```

Comment:
I think "step-container" class is now obsolete. If so, please remove it from layout.scss also.